group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorShard.java

```diff
@@ -121,6 +126,20 @@ public CoordinatorShardBuilder<GroupCoordinatorShard, Record> withTimer(
             return this;
         }
 
+        @Override
+        public CoordinatorShardBuilder<GroupCoordinatorShard, Record> withCoordinatorMetrics(
+            CoordinatorMetrics coordinatorMetrics
+        ) {
+            this.coordinatorMetrics = coordinatorMetrics;
+            return this;
+        }
+
+        @Override
```

Comment:
for my understanding, this topic partition is only used for metrics tagging?

Reply:
yes